### PR TITLE
Add missing dependencies to the README

### PR DESCRIPTION
**Pull request by amilcarlucas**
I found these using a clean docker with Ubuntu 20.04 LTS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,30 @@ This code has been tested in [Mbed v6.8](https://os.mbed.com/docs/mbed-os/v6.8),
 
 ## Dependencies
 
-This component needs `colcon` and other Python 3 packages in order to build micro-ROS packages:
+On a non ARM machine you will need to install the ARM cross compiler toolchain:
```
Comment (pablogs9) [inline README.md:7]: In an ARM machine without developers tool you will need to do so too.

File: README.md [hunk 2]
```diff
@@ -4,18 +4,30 @@ This code has been tested in [Mbed v6.8](https://os.mbed.com/docs/mbed-os/v6.8),
...
 
 ```bash
-pip3 install catkin_pkg lark-parser empy colcon-common-extensions
+sudo apt install arm-none-eabi-g++
```
Comment (pablogs9) [inline README.md:10]:
```suggestion
sudo apt install gcc-arm-none-eabi
```

like in https://github.com/micro-ROS/micro_ros_mbed/blob/6209542e8738509dbef5f54f0c4bc61e6ad2ec45/.github/workflows/ci.yml#L34

**Review by pablogs9 — CHANGES_REQUESTED** [review]

**Comment by pablogs9** [comment]
@mergify backport main

**Comment by amilcarlucas** [comment]
Squashed and fixed.

**Pull request merged**